packages/blade/.storybook/react/manager.js

```diff
@@ -22,7 +23,7 @@ export const theme = {
   fontCode: 'monospace',
 
   // Text colors
-  textColor: '#333333',
+  textColor: 'rgb(19, 38, 68)',
```

Comment:
our actual default text color from blade

Reply:
oh I tried but those `~` aliases from codebase are failing on storybook. I tried adding the aliases to storybook's webpack as well but its still failing to resolve certain files that way 🤔